src/components/Tabs/index.tsx

```diff
@@ -25,13 +25,17 @@ const Tabs = ({ tabs, onTabChange, activeTab }: TabsProps): JSX.Element => {
   const { isMobile } = useDeviceInfo();
 
   const tabStyles = {
-    fontSize: '14px',
+    color: '#7F775B',
```

Comment:
`theme.palette.TwClrTxtSecondary` is currently `#645E48` in the theme within the repo for some reason, but it looks like the value is `#7F775B` in the designs. I tried to regenerate the style dictionary and there was no change. There must be something out of date/sync with the styles dictionary system.

Reply:
Oh interesting, it looks like I missed the color switch from `$tw-clr-base-gray-600` → `$tw-clr-base-gray-500` for `$tw-clr-txt-secondary` during the Kaloko theme update.

Let's keep the change to `theme.palette.TwClrTxtSecondary` since `#645E48` and `#7F775B` is just one shade different. Thank you!

I'll update the `$tw-clr-txt-secondary` definition to `$tw-clr-base-gray-500` separately.